openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.upgrade.clicked.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
+  "type": "NavigationEvent",
+  "actor": {
+    "type": "Person",
+    "id": "edx.devstack.lms:18000/u/honor"
+  },
+  "action": "NavigatedTo",
+  "object": {
+    "id": "http://edx.devstack.lms:18000/dashboard",
+    "name": "Open edX Enrollment Upgrade"
+  },
+  "eventTime": "2018-11-01T07:11:21.947626",
+  "extensions": {
+    "extra_fields": {
+      "user_id": 7,
+      "org_id": " ",
+      "event_type": "edx.course.enrollment.upgrade.clicked",
+      "name": "edx.course.enrollment.upgrade.clicked",
+      "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+      "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+      "course_id": "",
+      "host": "tasawernawaz.sandbox.edx.org",
+      "path": "/event",
+      "username": "honor",
+      "event_source": "browser",
+      "ip": "116.58.62.58",
+      "event": "{\"location\": \"learner_dashboard\"}"
+    }
+  },
+  "referrer": "https://tasawernawaz.sandbox.edx.org/dashboard",
+  "session": "f3029807d5ac0b71b59b53e9e789f0fc"
```

Comment:
The `session` is a caliper `Entity`. and our current value of `session` does not comply to caliper. Please put this in the extra_fields section.